### PR TITLE
Enable DH Tekton plugin

### PR DESCRIPTION
**Pull request by Roming22**

### CODE REVIEW

File: test/data/helm-chart/template.yaml [hunk 1]
```diff
@@ -553,6 +401,275 @@ spec:
...
+              curl --fail --silent --show-error --location \
+                https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3 \
+                  | bash
+        
+              YQ_VERSION="v4.40.5"
+              curl --fail --location --output "/usr/bin/yq" --silent --show-error "https://github.com/mikefarah/yq/releases/download/${YQ_VERSION}/yq_linux_amd64"
+              chmod +x "/usr/bin/yq"
+        
+              CHART="rhtap"
+              NAMESPACE="rhtap"
```
Comment (otaviof) [inline test/data/helm-chart/template.yaml:426]: Let's use the namespace defined by Helm, i.e.:

https://github.com/redhat-appstudio/rhtap-installer/blob/628a45c1d8ca383bdac34c33975552d5d1ba9134/templates/trusted-profile-analyzer/includes/_configure.tpl#L33-L34

File: test/data/helm-chart/template.yaml [hunk 2]
```diff
@@ -553,6 +401,275 @@ spec:
...
+                            - config:
+                                if:
+                                  allOf:
+                                  - isTektonCIAvailable
+                                layout:
+                                  gridColumn: 1 / -1
+                                  gridRowStart: 1
+                              importName: TektonCI
+                              mountPoint: entity.page.ci/cards
+              EOF
```
Comment (otaviof) [inline test/data/helm-chart/template.yaml:509]: Should we store these lines as a standalone template file?

File: test/data/helm-chart/template.yaml [hunk 3]
```diff
@@ -553,6 +401,275 @@ spec:
...
+                    webhookUrl: GITHUB__APP__WEBHOOK_URL
+                  host: github.com
+              techdocs:
+                builder: local
+                generator:
+                  runIn: local
+                publisher:
+                  type: local
+              _EOF_
+              echo -n "."
```
Comment (otaviof) [inline test/data/helm-chart/template.yaml:613]: That's another snippet I think we should consider moving to a standalone template file. And also, how the environment variables, like `${GITHUB__APP__PRIVATE_KEY}` are substituted?

File: test/data/helm-chart/template.yaml (hunk 2 above)
Comment (Roming22) [reply test/data/helm-chart/template.yaml:509]: This file is just a rendering of the helm chart. It's there as a summary of the changes to the chart's behavior.

That file is not used by the chart at all.

File: test/data/helm-chart/template.yaml (hunk 3 above)
Comment (Roming22) [reply test/data/helm-chart/template.yaml:613]: See my previous comment.

File: test/data/helm-chart/template.yaml (hunk 1 above)
Comment (Roming22) [reply test/data/helm-chart/template.yaml:426]: This file is just a rendering of the helm chart.  The source has the Helm defined value.

**Pull request merged**